uniffi/src/bindings/kotlin/mod.rs

```diff
@@ -56,10 +48,8 @@ pub fn compile_kotlin_bindings(ci: &ComponentInterface, out_dir: &OsStr) -> Resu
         .arg(&kt_file)
         .arg("-d")
         .arg(jar_file)
-        .spawn()
-        .map_err(|_| anyhow::anyhow!("failed to spawn `kotlinc`"))?
-        .wait()
-        .map_err(|_| anyhow::anyhow!("failed to wait for `kotlinc` subprocess"))?;
+        .spawn()?
+        .wait()?;
```

Comment:
Turns out I had a while bunch of spurious `map_err` throughout the codebase; `std::io::Error` is `Into<anyhow::Error>` so they're not needed.

Reply:
I’m wondering if this will still give us nice context for errors—“failed to wait for `kotlinc` subprocess” pinpoints the problem much better than some generic “process exited with status code 1337”. Maybe Anyhow’s [`context` and `with_context`](https://docs.rs/anyhow/1.0.13/anyhow/trait.Context.html) are the ticket there. Not saying we need to sprinkle those everywhere, just that I liked your old error messages too! 😸